redis/src/main/scala/zio/redis/api/Cluster.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2021 John A. De Goes and the ZIO contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.redis.api
+
+import zio.redis.Input._
+import zio.redis.Output.{ChunkOutput, ClusterPartitionOutput, UnitOutput}
+import zio.redis._
+import zio.redis.api.Cluster.{AskingCommand, ClusterSetSlots, ClusterSlots}
+import zio.redis.options.Cluster.SetSlotSubCommand._
+import zio.redis.options.Cluster.{Partition, Slot}
+import zio.{Chunk, ZIO}
+
+trait Cluster {
+
+  def asking: ZIO[Redis, RedisError, Unit] =
+    AskingCommand.run(())
+
+  def slots: ZIO[Redis, RedisError, Chunk[Partition]] = {
+    val command = RedisCommand(ClusterSlots, NoInput, ChunkOutput(ClusterPartitionOutput))
+    command.run(())
+  }
+
+  def setSlotStable(slot: Slot): ZIO[Redis, RedisError, Unit] = {
```

Comment:
Not sure if we should have these as separate functions, but if I'm not mistaken, we're already doing that elsewhere, so it's consistent :+1:

Reply:
Yeah, we have  examples where method that can have different behavior depending on it's parameters in redis, we split by different methods in client